javascript/hyper-express/app.js

```diff
@@ -0,0 +1,25 @@
+const HyperExpress = require('hyper-express');
+
+// Create a new instance of HyperExpress HTTP
+const port = 3000;
+const app = new HyperExpress.Server();
+
+// GET "/" => 200 with empty body
+app.get('/', (request, response) => {
+    response.send('');
+});
+
+// GET "/user/:id" => 200 with "id" as body
+app.get('/user/:id', (request, response) => {
+    response.send(request.path_parameters.id);
+});
+
+// POST "/user" => 200 with empty body
+app.post('/user', (request, response) => {
+    response.send('');
+});
+
+// Start the server on port 3000
+app.listen(port)
+    .then(() => console.log(`Started HyperExpress on port ${port}`))
```

Comment:
I think this affect `req/s` show here

Reply:
I just removed the .then()/.catch() since under the hood it calls the uWS.listen() method anyways which is exactly what all of the other uWebsockets.js webservers will do as well.